src/store/modules/job/actions.ts

```diff
@@ -286,31 +286,40 @@ const actions: ActionTree<JobState, RootState> = {
       // getting job with status Service draft as well, as this information will be needed when scheduling
       // a new batch
       // TODO: this needs to be updated when we will be storing the draft and pending jobs separately
-      const batchBrokeringJobs = [] as any
-      const batchBrokeringJobEnum = (JSON.parse(process.env.VUE_APP_ODR_JOB_ENUMS as string) as any)['BTCH_BRKR_ORD']
-      resp.data.docs.filter((job: any) => job.systemJobEnumId === batchBrokeringJobEnum).map((job: any) => {
-        batchBrokeringJobs.push({
-          ...job,
-          id: job.jobId,
-          frequency: job.tempExprId,
-          enumId: job.systemJobEnumId,
-          status: job.statusId
-        })
+      const batchJobEnums = JSON.parse(process.env?.VUE_APP_BATCH_JOB_ENUMS as string)
+      let batchJobEnumIds = Object.values(batchJobEnums)?.map((job: any) => { 
+        return job.id;
```

Comment:
We can remove the return statement and instead just directly use job.id

Reply:
Fixed it as your directed sir.